src/MassTransit.SignalR/MassTransit.SignalR.csproj

```diff
@@ -14,8 +14,15 @@
     <Description>MassTransit SignalR Backplane support; $(Description)</Description>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net6.0'">
```

Comment:
Does anything other than .net6.0 still make sense for consuming SignalR? `Microsoft.AspNetCore.SignalR.Core` is 3.5+ years old, deprecated, and has transitive dependencies with security warnings. If not, I can shrink the target frameworks.

Reply:
I'm fine if you want to make it net6 only.